pkg/ingester/ingester_v2.go

```diff
@@ -2040,7 +2067,15 @@ func (i *Ingester) v2FlushHandler(w http.ResponseWriter, _ *http.Request) {
 		}
 
 		level.Info(i.logger).Log("msg", "flushing TSDB blocks: finished")
-	}()
+	}
+
+	const waitParam = "wait"
```

Comment:
[nit] I would define it close to the tenant param (if we'll keep it), given they both apply to the same API endpoint.

Reply:
I suggest we keep it. Moving to `tenantParam`.